content/src/scripts/containers/Filter/Filter.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import {
+  isMergeRequestShown
+} from "../../../../../event/src/actions/API";
+
+import "./styles.css";
+
+
+function Filter({
+
+}) {
+
+  // const [checkedTests, setCheckedTest] = useState();
+  // useEffect(() => {
+  //   handleChange = () => {
+  //     setCheckedTest(!checkedTests, () => {
+  // window.location.reload();
+  //     });
+  //   };
+  // }, []);
+
+  return (
+    <div className="spantree-filter-body">
+      {isMergeRequestShown() ? (
+        <div className="spantree-filter-header">
+          <input type="checkbox" id="filterTests" name="tests" />
+          {/* <input type="checkbox" id="filterTests" name="tests" onClick={handleChange} defaultChecked={checkedTests} /> */}
```

Comment:
I could not get it to work that the checkbox state is properly stored and the checkbox is checked... the idea of these checkboxes is to filter out unwanted changes from the flat tree of changes